pipelines/release.yml

```
@@ -85,6 +85,7 @@ groups:
   - bump-cbd-versions
   - patch
   - discover-component-version
+  - update-images
```

guessing this is a typo/old name:

```suggestion
  - rebuild-images
```